CougSat1-PMIC/src/components/PowerNodeDirect.h

```diff
@@ -22,13 +22,15 @@
 #include "drivers/LTC2499.h"
 #include "mbed.h"
 
-class PowerNodeDirect : public PowerNode {
+class PowerNodeDirect : public PowerNode 
+{
 public:
-  PowerNodeDirect(LTC2499 &adc, LTC2499Channel_t channel, double shunt,
+  PowerNodeDirect(LTC2499 & adc, LTC2499Channel_t channel, double shunt,
```

Comment:
These constructors need the priority and expected current added to the constructor. You should have a compilation error currently